sklearn/linear_model/logistic.py

```diff
@@ -1897,7 +1897,7 @@ class LogisticRegressionCV(LogisticRegression, BaseEstimator,
     >>> from sklearn.datasets import load_iris
     >>> from sklearn.linear_model import LogisticRegressionCV
     >>> X, y = load_iris(return_X_y=True)
-    >>> clf = LogisticRegressionCV(cv=5, random_state=0).fit(X, y)
+    >>> clf = LogisticRegressionCV(Cs=6, cv=5, random_state=0).fit(X, y)
```

Comment:
This may be a bit mysterious to the reader. Why 6?

Reply:
Yes, I suppose readability in the docs is more important than run time. Reverted. 6 was just a number smaller than 10 (default). Actually `solver="liblinear"` would have been even faster as lbfgs takes a bit longer to converge on such small datasets.